src/main/java/no/unit/nva/model/instancetypes/NonPeerReviewed.java

```diff
@@ -12,9 +14,14 @@
      * @param peerReviewed a boolean value.
      * @throws UnexpectedException thrown if the boolean equals true.
      */
-    public void setPeerReviewed(boolean peerReviewed) throws UnexpectedException {
+    public final void setPeerReviewed(boolean peerReviewed) throws UnexpectedException {
```

Comment:
This is not actually setting the field even if it is false?

Reply:
I think that this made sense at one point, I don't think it does following refactoring. Will fix.

Reply:
There is no field to set. `isPeerReviewed()` returns always false, so there is no reason for a "reviewed" field to exist. I think that the exception has the meaning that the input data are invalid